oak_restricted_kernel/src/interrupts.rs

```diff
@@ -296,27 +286,13 @@ extern "x86-interrupt" fn machine_check_handler(stack_frame: InterruptStackFrame
     shutdown::shutdown();
 }
 
-bitflags! {
```

Comment:
Please don't :)

I created this bitfield so that the error flags would be easily readable. Could you replace them with the constant values?

Reply:
Done.